playground/frontend/lib/pages/playground/components/editor_textarea_wrapper.dart

```diff
@@ -104,11 +107,8 @@ class CodeTextAreaWrapper extends StatelessWidget {
                             AnalyticsService analyticsService =
                                 AnalyticsService.get(context);
                             final stopwatch = Stopwatch()..start();
-                            final exampleName = getAnalyticsExampleName(
-                              state.selectedExample,
-                              state.isExampleChanged,
-                              state.sdk,
-                            );
+                            final exampleName = getAnalyticsExampleName(state);
```

Comment:
This method is changed to only accept `PlaygroundState` for the following reasons:
- In both its uses, all the properties are taken from `PlaygroundState` anyway.
- It's not a widget's business to know what exact properties form the name for analytics.
- This decoupling will allow to easily change how we produce the name for analytics.